MaxText/train.py

```diff
@@ -480,14 +482,28 @@ def main(argv: Sequence[str]) -> None:
   config = pyconfig.config
   validate_train_config(config)
   os.environ["TFDS_DATA_DIR"] = config.dataset_path
+
+  if config.create_vertex_tensorboard and jax.process_index()==0:
```

Comment:
Let's encapsulate this in a function for readability. Happy for that function to go in train.py or elsewhere.